pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -206,9 +206,7 @@ func manageKubeAPIServerConfig(client coreclientv1.ConfigMapsGetter, recorder ev
 	configMap := resourceread.ReadConfigMapV1OrDie(v410_00_assets.MustAsset("v4.1.0/kube-apiserver/cm.yaml"))
 	defaultConfig := v410_00_assets.MustAsset("v4.1.0/config/defaultconfig.yaml")
 	configOverrides := v410_00_assets.MustAsset("v4.1.0/config/config-overrides.yaml")
-	specialMergeRules := map[string]resourcemerge.MergeFunc{
-		".oauthConfig": RemoveConfig,
-	}
+	specialMergeRules := map[string]resourcemerge.MergeFunc{}
```

Comment:
why this?

It would deserve its own commit with an explanation.

Reply:
The observed config is going to appear in this path. Boy do I hope that there are no side effects to this in KAS and that we cleaned all of the code that should have been dead in 4.x.